app/templates/mandatarissen/persoon/mandataris-rework.hbs

```diff
@@ -0,0 +1,116 @@
+{{breadcrumb
+  this.model.mandataris.bekleedt.bestuursfunctie.label
+  route="mandatarissen.persoon.mandataris"
+  model=this.model.mandataris.id
+}}
+
+<div class="au-o-grid au-u-padding">
+  <div class="au-o-grid__item au-u-1-2">
+    <p class="au-u-h2 au-u-medium">
+      {{this.bestuursorganenTitle}}
+    </p>
+    <p class="au-u-h3">
+      {{this.persoon.gebruikteVoornaam}}
+      {{this.persoon.achternaam}}
+    </p>
+  </div>
+  <div class="au-o-grid__item au-u-1-2 au-u-text-left">
+    <div class="au-u-flex au-u-flex--end au-u-margin-bottom">
+      {{#if this.notOwnedByUs}}
+        <AuAlert @skin="info" @icon="eye" @closable={{false}}>
+          <div class="au-u-flex au-u-flex--spaced-tiny">
+            <span>Beheerd door:</span>
+            <Mandaat::OwnershipTags @owners={{this.model.owners}} />
+          </div>
+          <p>Je kan deze mandataris niet bewerken.</p>
+        </AuAlert>
+      {{else}}
+        <AuButtonGroup>
+          <Shared::Tooltip
+            @showTooltip={{true}}
+            @tooltipText={{this.toolTipText}}
```

Comment:
shouldn't show the tooltip about the prep of the next period if that prep is already done